addon/mixins/numeric.js

```diff
@@ -49,8 +49,8 @@ export default Mixin.create({
     }
 
     if (this.options.range && isArray(this.options.range)) {
-      first = this.options.range[0];
-      last = this.options.range[1];
+      let first = this.options.range[0];
+      let last = this.options.range[1];
```

Comment:
@alexdiliberto fyi, we could use array destructuring assignment here for conciseness:

`let [ first, last ] = this.options;`